lib/tasks/data/sanitize_data.rake

```diff
@@ -0,0 +1,56 @@
+require 'highline'
+
+namespace :ofn do
+  namespace :data do
+    desc 'Sanitize data'
+    task sanitize: :environment do
+      guard_and_warn
+
+      Spree::User.update_all("email = concat(id, '_ofn_user@example.com'),
+                              login = concat(id, '_ofn_user@example.com'),
+                              unconfirmed_email = concat(id, '_ofn_user@example.com')")
+      Spree::Customer.update_all("email = concat(id, '_ofn_customer@example.com'),
+                                  name = concat('Customer Number ', id)")
+      Spree::Order.update_all("email = concat(id, '_ofn_order@example.com')")
+      Spree::Address.update_all("
+        firstname = concat('Ms. Number', id), lastname = 'Jones',  phone = '01234567890',
+        alternative_phone = '01234567890', address1 = 'Dummy address',
+        address2 = 'Dummy address continuation', city = 'Dummy City', zipcode = '0000',
```

Comment:
Do you think it would be reasonable to keep the city and zipcode? That would avoid the map from displaying all in one spot.

Reply:
yes, done.